org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/DebugUIPlugin.java

```diff
@@ -1252,7 +1252,7 @@ public void done(IJobChangeEvent event) {
 		job.setName(MessageFormat.format(DebugUIMessages.DebugUIPlugin_25, new Object[] {configuration.getName()}));
 
 		if (wait) {
-			progressService.showInDialog(workbench.getActiveWorkbenchWindow().getShell(), job);
+			progressService.showInDialog(workbench.getDisplay().getActiveShell(), job);
```

Comment:
This shell can be null, not sure if that's an issue.

Reply:
> This shell can be null, not sure if that's an issue.

It will not, but I saw the code in same class uses getShell() so better to use that for consistency